### PR TITLE
TASK-935: Callback server failhard squash

### PR DESCRIPTION
**Pull request by arfathpasha**

### CODE REVIEW

**Comment by MrCreosote** [comment]
Not sure why, but there's a couple of commits in this PR that shouldn't be in here

File: src/java/us/kbase/mobu/runner/ModuleRunner.java [hunk 1]
```diff
@@ -239,12 +240,9 @@ public void logNextLine(String line, boolean isError) {
...
             jettyServer.start();
         } else {
-            if (callbackNetworks != null && callbackNetworks.length > 0) {
-                throw new IllegalStateException("No proper callback IP was found, " +
-                        "please check callback_networks parameter in configuration");
-            }
-            System.out.println("WARNING: No callback URL was received " +
-                    "by the job runner. Local callbacks are disabled.");
+            throw new IllegalStateException("No callback URL was received " +
+                    "by the job runner. Local callbacks are disabled. " +
```
Comment (MrCreosote) [inline src/java/us/kbase/mobu/runner/ModuleRunner.java]: This error message doesn't seem right.

File: src/java/us/kbase/mobu/tester/ModuleTester.java [hunk 2]
```diff
@@ -212,12 +213,9 @@ public void logNextLine(String line, boolean isError) {
...
             jettyServer.start();
         } else {
-            if (callbackNetworks != null && callbackNetworks.length > 0) {
-                throw new IllegalStateException("No proper callback IP was found, " +
-                		"please check callback_networks parameter in test.cfg");
-            }
-            System.out.println("WARNING: No callback URL was received " +
-                    "by the job runner. Local callbacks are disabled.");
+            throw new IllegalStateException("No callback URL was received " +
+                    "by the job runner. Local callbacks are disabled. " +
```
Comment (MrCreosote) [inline src/java/us/kbase/mobu/tester/ModuleTester.java]: Same

**Comment by arfathpasha** [comment]
I found that strange too. I fetched recent changes, did a merge and then PR. But it looks like what I fetched became part of the PR. The PR step strangely didn't complain about merge conflicts despite the fetch becoming part of the PR. I have not seen this happen before and I am not sure what I did wrong to make it happen.

**Pull request closed without merging**